src/pkg/scan/dao/scanner/registration.go

```diff
@@ -123,10 +123,18 @@ func ListRegistrations(ctx context.Context, query *q.Query) ([]*Registration, er
 	}
 
 	// Order the list
-	if query.Sorting != "" {
-		qs = qs.OrderBy(query.Sorting)
+	if len(query.Sorts) > 0 {
```

Comment:
I think the following logic is duplicated because when call the `orm.QuerySetter`, there will be a common `setSorts` function to handle it. https://github.com/goharbor/harbor/blob/0da13ebd28d702e35751860327b8dec38a313ad0/src/lib/orm/query.go#L153

Reply:
So, the `Query Setter` is already returning in sorted order and we just have to replace the explicit Sorting from the code?

Reply:
Yes, since there is a common logic processing, there is no need for redundant processing here.